doc/apiary/v2/fiware-ngsiv2-reference.apib

```diff
@@ -713,8 +713,65 @@ so notification receiver can know the format used without needing processing not
 
 ### Notification templates
 
-Based on the "template" field defined as part of the subscription and in a macro replacement language
-that could be used in "payload", "url", "headers", "method" and "qs". Details TBD.
+At notification time, NGSIv2 implementations will send a notification based on the `notification.httpExtended`
```

Comment:
Better not to use 'will' (and plural):

    At notification time, NGSIv2 implementations send notifications based on